spring-cloud-gcp-samples/spring-cloud-gcp-pubsub-stream-dead-letter-sample/src/main/java/com/example/SinkExample.java

```diff
@@ -63,4 +66,9 @@ public Consumer<UserMessage> deadLetterMessages() {
             userMessage.getUsername(),
             userMessage.getBody());
   }
+
+  @Bean
+  public JsonMessageConverter customJsonMessageConverter(ObjectMapper objectMapper) {
```

Comment:
Can we add a comment here pointing to either our bug or https://github.com/spring-cloud/spring-cloud-stream/issues/2977 ?